config/mongoid.yml

```diff
@@ -1,27 +1,23 @@
 development:
-  sessions:
+  clients:
     default:
       uri: <%= ENV["MONGODB_URI"] || "mongodb://localhost/govuk_assets_development" %>
       options:
-        write:
-          w: 1
-        read: primary
+  options:
 test:
-  sessions:
+  clients:
     default:
       uri: <%= ENV["TEST_MONGODB_URI"] || "mongodb://localhost/govuk_assets_test" %>
       options:
-        write:
-          w: 1
-        read: primary
-        max_retries: 1
-        retry_interval: 0
+        read:
+          mode: :primary
+        max_pool_size: 1
 production:
-  sessions:
+  clients:
     default:
       uri: <%= ENV['MONGODB_URI'] %>
       options:
-        write:
-          w: 1
-        read: primary
-        retry_interval: 120
+        read:
+          mode: :primary
+  options:
+    use_activesupport_time_zone: true
```

Comment:
Minor: It seems slightly odd that this option is only included in the `production` environment. Do you know why it's not included in `development` or `test`?